Cargo.toml

```diff
@@ -31,3 +31,4 @@ base64 = "0.13"
 stringprep = "0.1"
 tokio-rustls = "0.23"
 rustls-pemfile = "1"
+wg = "0.1.0"
```

Comment:
Based on [this](https://github.com/al8n/wg/), it doesn't look like a well-known crate. Are we sure we need it?